src/app/components/JumpTo/index.tsx

```diff
@@ -1,28 +1,46 @@
+/* eslint-disable react-hooks/rules-of-hooks */
 /** @jsx jsx */
-import { useContext } from 'react';
+import { useContext, useState, useEffect } from 'react';
 import { jsx } from '@emotion/react';
 import { ServiceContext } from '#contexts/ServiceContext';
 import useViewTracker from '#app/hooks/useViewTracker';
 import useClickTrackerHandler from '#app/hooks/useClickTrackerHandler';
 import { EventTrackingMetadata } from '#app/models/types/eventTracking';
 import Text from '#app/components/Text';
-import InlineLink from '#app/components/InlineLink';
+import isLive from '#app/lib/utilities/isLive';
 import idSanitiser from '../../lib/utilities/idSanitiser';
+import styles from './index.styles';
 
 export interface JumpToProps {
   jumpToHeadings?: Array<{ heading: string }>;
-  eventTrackingData?: EventTrackingMetadata;
 }
 
-const JumpTo = ({ jumpToHeadings, eventTrackingData }: JumpToProps) => {
+const eventTrackingData: EventTrackingMetadata = {
+  componentName: 'jumpto',
+};
```

Comment:
@LilyL0u I changed this here as well, but happy to wait for your changes to merge in as they'll be changing it to the same thing!